R/plots.R

```diff
@@ -364,18 +366,30 @@ plot_marginals <- function(x, metric = NULL) {
 
   x <-
     x %>%
-    dplyr::select(dplyr::one_of(param_cols), mean, .metric) %>%
+    dplyr::rename(resamples = n) %>%
```

Comment:
I think my only comment is that this might be _slightly_ clearer if this was `n_resamples`?